src/integrationTest/java/uk/gov/hmcts/reform/iacaseapi/docs/SwaggerPublisher.java

```diff
@@ -45,6 +46,12 @@ public void shouldGenerateDocs() throws Exception {
 
         LOG.info("Generating Swagger Docs");
 
+        File linuxTmpDir = new File("/tmp");
+        boolean linuxTmpDirExists = linuxTmpDir.exists();
+        if (!linuxTmpDirExists) {
```

Comment:
I think it reads just as well inlined and with one less line of code, and I got to write a review comment, yay!